axolotl-web/src/pages/Register.vue

```diff
@@ -126,6 +127,10 @@ export default {
     justify-content:center;
     align-items:center;
   }
+  .vue-phone-number-input {
+    background-color: $input-bg;
```

Comment:
The rendering here is definitely **not** perfect, but at this point, the user shouldn't have been able to switch the theme anyway